index.js

```diff
@@ -49,3 +49,21 @@ function compose (middleware) {
     }
   }
 }
+
+/**
+ * composes a wrapped middleware using middleware._wrappers
+ *
+ * @param {Function} middleware
+ * @return {Function} composed
+ * @api public
+ */
+
+function wrap (middleware) {
+  if (!middleware) return false
+  if (!middleware._wrappers) return middleware
+  if (!Array.isArray(middleware._wrappers)) throw new TypeError('._wrappers must be an array')
+  if (!middleware._wrappers.length) return middleware
+  const wrappers = middleware._wrappers
+  delete middleware._wrappers
+  return compose([...wrappers, middleware])
```

Comment:
Also, using `...` for array concatenation isn't supported on Node v4 (causing the tests to fail).